tests/drivers/build_all/sensor/prj.conf

```diff
@@ -3,6 +3,7 @@ CONFIG_TEST_USERSPACE=y
 CONFIG_LOG=y
 CONFIG_SENSOR_LOG_LEVEL_DBG=y
 CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_ADC=y
```

Comment:
Is this required since `ADC` is auto-selected by `MCP970X`?

Reply:
Not really but other sensors are selecting these auto selected the Kconfig symbols so maintained that pattern.  I can possibly simplify this file and remove these already selected symbols.